### PR TITLE
WIP: Linter documentation

### PR DESCRIPTION
**Pull request by dav1x**

### CODE REVIEW

File: creating/creating_index.adoc [hunk 1]
```diff
@@ -491,8 +491,53 @@ include::help.adoc[]
 
 === Verifying Dockerfile (linter)
 
+The Dockerfile-lint is a rule based 'linter' for verifying Dockerfiles. The rules are used to check syntax and best practice options for things such as 'Was yum clean up evoked after a package installation?' or 'In the RUN section did the writer link commands via semicolons or double ampersands?'. The input rules are called via a set of yaml files. At the time of this writing, there are a number of templates from base to automated build configurations.
```
Comment (baude) [inline creating/creating_index.adoc]: Break to list?

s/called/defined

File: creating/creating_index.adoc [hunk 2]
```diff
@@ -491,8 +491,53 @@ include::help.adoc[]
...
+The Dockerfile-lint is a rule based 'linter' for verifying Dockerfiles. The rules are used to check syntax and best practice options for things such as 'Was yum clean up evoked after a package installation?' or 'In the RUN section did the writer link commands via semicolons or double ampersands?'. The input rules are called via a set of yaml files. At the time of this writing, there are a number of templates from base to automated build configurations. 
 
+There are two iterations of the linter. 
 
+The CLI version of the project is available here:
+link:https://github.com/projectatomic/dockerfile_lint[Dockerfile_lint] 
+
+The online version of the project is available here:
+link:https://access.redhat.com/labs/linterfordockerfile/#/[Online linter]
+
```
Comment (baude) [inline creating/creating_index.adoc:701]: ==== Installing the linter

File: creating/creating_index.adoc [hunk 3]
```diff
@@ -491,8 +491,53 @@ include::help.adoc[]
...
+cd dockerfile_lint
+npm install
+```
+
+Upon evokation of the CLI version of the linter, some sample output may include:
+
+```
+-> bin/dockerfile_lint -f ~/Documents/manage-iq/Dockerfile.2 -r sample_rules/basic_rules.yaml 
+
+--------ERRORS---------
```
Comment (baude) [inline creating/creating_index.adoc]: use ``` to differentiate between text and usual stdout output

File: creating/creating_index.adoc [hunk 4]
```diff
@@ -491,8 +491,53 @@ include::help.adoc[]
...
+The online version of the project is available here:
+link:https://access.redhat.com/labs/linterfordockerfile/#/[Online linter]
+
+For the next section, we will assume that the linter will be installed manually via npm using the following commands: 
+
+```
+git clone https://github.com/projectatomic/dockerfile_lint/
+cd dockerfile_lint
+npm install
+```
```
Comment (baude) [inline creating/creating_index.adoc:708]: Perhaps a short section showing users where the rules are ...

File: creating/creating_index.adoc [hunk 5]
```diff
@@ -491,8 +491,53 @@ include::help.adoc[]
...
+Reference -> https://docs.docker.com/reference/builder/#user
+
+
+
+--------INFO---------
+
+INFO: There is no 'ENTRYPOINT' instruction. None. 
+Reference -> https://docs.docker.com/reference/builder/#entrypoint
+
+```
```
Comment (baude) [inline creating/creating_index.adoc:770]: Discuss the difference between and error or warning?

Comment (baude) [inline creating/creating_index.adoc:770]: Discuss how they might use your base rules with a custom set?

File: creating/creating_index.adoc (hunk 4 above)
Comment (dav1x) [inline creating/creating_index.adoc:708]: Added this and organized the information under sub headers for creating parsing rules, Using the linter and installing the linter

**Pull request merged**